plugins/fixtures/index.js

```diff
@@ -0,0 +1,56 @@
+const Boom = require('boom');
+const Hoek = require('hoek');
+const fetch = require('node-fetch');
+
+module.exports = {
+  name: 'q-fixtures',
+  dependencies: 'q-base',
+  register: async function (server, options) {
+    Hoek.assert(server.settings.app.tools && typeof server.settings.app.tools.get === 'function', new Error('server.settings.app.tools.get needs to be a function')); 
+
+    // get a list of already existing (stored in db) and new fixture data
+    server.method('fixtures.get', async () => {
```

Comment:
the method should be named `plugins.q.fixtures.get`. There is the unwritten convention that only core plugins are allowed to register server methods without `plugins.q.<fixture-name>` prefix.